cartography/intel/github/util.py

```diff
@@ -81,6 +81,12 @@ def fetch_all(token, api_url, organization, query, resource_type, field_name):
                 resource_type,
             )
             break
+        except requests.exceptions.HTTPError as e:
```

Comment:
what are all the call patterns for these?  

Breaking instead of outright failing has caused problems in the past for ACL.  I in general prefer to fail outright but it depends on the call patterns

Reply:
Our sync for GitHub objects is best effort and until we get DAGs working or a more robust retry mechanism, I'd rather not let a GitHub failure cause the rest of the sync to end.

I agree with the whole 'fail-loudly' idea, but for now until we build configurable error handling, I'd prefer to 'warn-loudly' that there was missing data, load what we have, and then the caller can manually do a re-run.